src/helpers/OTHelper.js

```diff
@@ -19,19 +19,26 @@ const reassignEvents = (type, customEvents, events) => {
 
 const sanitizeBooleanProperty = property => (property || property === undefined ? true : property);
 
-const logOT = (apiKey, sessionId) => {
+const getLogBody = (apiKey, sessionId, action, connectionId) => {
```

Comment:
I was thinking HTML body here , when I first read it.
Can we just say getLog?

Reply:
updated in latest commit